deep_groove_project/src/first_eda/first_eda.py

```diff
@@ -42,10 +43,10 @@ def show_3d_scatter(df):
     n = df.iloc[:, 1]
     lifetime = df.iloc[:, 2]
 
-    ax.scatter(fr, n, lifetime)
+    ax.scatter(fr, n, np.log(lifetime))
```

Comment:
I would like to use the log with base 10 here

https://numpy.org/doc/stable/reference/generated/numpy.log10.html#numpy.log10

The natural logarithm does not provide a good base from a engineering perspective.

Reply:
changes it to log10(Lifetime)